src/utils/validateIdentifierInputs.ts

```diff
@@ -28,7 +28,7 @@ export const validateIdentifierInputs = (
     if (
       textInputType === "PAYMENT_RECEIPT" &&
       !isMatchRegex(
-        value,
+        value.trim(),
```

Comment:
noticed we are calling `trim()` multiple times across this function, do you think it's better to trim the value at the start?

Reply:
updated to trim all values from the start!